src/popup/components/TransactionList.vue

```diff
@@ -10,6 +10,10 @@
         v-for="transaction in filteredTransactions"
         :key="transaction.hash"
         :transaction="transaction"
+        :confirmed-by="getConsensusInfo(transaction).confirmedBy"
+        :confirmations-required="getConsensusInfo(transaction).confirmationsRequired"
+        :total-signers="getConsensusInfo(transaction).totalSigners"
+        :is-multisig-transaction="getConsensusInfo(transaction).isMultisigTransaction"
         :data-cy="transaction.pending && 'pending-txs'"
```

Comment:
getConsensusInfo(transaction) -> to computed

Reply:
how?

Reply:
into TransactionItem